packages/pixeloven/cli-addon-storybook/src/types.ts

```diff
@@ -5,11 +5,24 @@ export enum StorybookExecutionType {
     start = "start",
 }
 
+export interface StorybookExecutionOptions {
+    outputDir?: string;
+    port?: number;
+    noQuiet?: boolean;
+}
+
+/* Note: keep this enum up to date matching the options interface above for runtime argument checking */
+export enum StorybookExecutionOptionTypes {
+    outputDir = "outputDir",
+    port = "port",
+    noQuiet = "noQuiet",
```

Comment:
I guess we should add `output-dir` and `no-quiet` here to keep with the kebab-case used in the other CLIs